source/includes/_wallet.md

```diff
@@ -1 +1,1029 @@
+mnemonic | String | | A mnemonic phrase to use to instantiate an hd private key. One will be generated if none provided
+
+
+## Wallet Auth
+> The following samples return a wallet object
+
+```shell--vars
+    token='977fbb8d212a1e78c7ce9dfda4ff3d7cc8bcd20c4ccf85d2c9c84bbef6c88b3c'
+    id='primary'
+```
+
+```shell--curl
+curl $url/wallet/$id/send \
+    -H 'Content-Type: application/json' \
+    -d '{ "token": "$token" ... }'
+```
+
+```shell--cli
+bcoin cli wallet get --network=testnet --token=$token
+```
+
+```javascript
+`use strict`
+
+const httpWallet = new bcoin.http.Wallet({ 
+    id: id, 
+    token: token,
+    network: 'testnet',
+    db: 'leveldb'
+});
+
+(async () => {
+  await httpWallet.open();
+  const wallet = await httpWallet.getInfo();
+  console.log(wallet);
+  httpWallet.close();
+})();
+```
+
+Individual wallets have their own api keys, referred to internally as "tokens" (a 32 byte hash - calculated as `HASH256(m/44'->ec-private-key|tokenDepth)`).
+
+A wallet is always created with a corresponding token. When using api endpoints
+for a specific wallet, the token must be sent back in the query string or json
+body.
+
+e.g. To get information from a wallet that requires a token
+
+`GET /wallet/primary/tx/:hash?token=977fbb8d212a1e78c7ce9dfda4ff3d7cc8bcd20c4ccf85d2c9c84bbef6c88b3c`
+
+
+## Get Wallet Info
+
+```shell--curl
+curl http://localhost:18332/wallet/test/
+
+```
+
+```shell--cli
+bcoin cli wallet get --id=test --network=testnet
+```
+
+```javascript
+`use strict`
+
+const client = new bcoin.http.Client({  network: 'testnet' });
+const id = 'test';
+
+(async () => {
+  const wallet = await client.getWallet(id);
+  console.log(wallet);
+})();
+```
+
+> Output is same as wallet object above
+
+```json 
+{
+  "network": "testnet",
+  "wid": 1,
+  "id": "test",
+  "initialized": true,
+  "watchOnly": false,
+  "accountDepth": 1,
+  "token": "2d04e217877f15ba920d02c24c6c18f4d39df92f3ae851bec37f0ade063244b2",
+  "tokenDepth": 0,
+  "state": {
+    "tx": 177,
+    "coin": 177,
+    "unconfirmed": "8150.0",
+    "confirmed": "8150.0"
+  },
+  "master": {
+    "encrypted": false
+  },
+  "account": {
+    "name": "default",
+    "initialized": true,
+    "witness": false,
+    "watchOnly": false,
+    "type": "pubkeyhash",
+    "m": 1,
+    "n": 1,
+    "accountIndex": 0,
+    "receiveDepth": 8,
+    "changeDepth": 1,
+    "nestedDepth": 0,
+    "lookahead": 10,
+    "receiveAddress": "mu5Puppq4Es3mibRskMwoGjoZujHCFRwGS",
+    "nestedAddress": null,
+    "changeAddress": "n3nFYgQR2mrLwC3X66xHNsx4UqhS3rkSnY",
+    "accountKey": "tpubDC5u44zLNUVo2gPVdqCbtX644PKccH5VZB3nqUgeCiwKoi6BQZGtr5d6hhougcD6PqjszsbR3xHrQ5k8yTbUt64aSthWuNdGi7zSwfGVuxc",
+    "keys": []
+  }
+}
+```
+
+Get wallet info by ID.
+
+### HTTP Request
+`GET /wallet/:id`
+
+Parameters | Description
+---------- | -----------
+id <br> _string_ | named id of the wallet whose info you would like to retrieve
+
+## Get Master HD Key
+```javascript
+  let id
```

Comment:
I used to use a style guide that didn't like this format. Seems the bcoin linter doesn't have an opinion. Is this your preferred way? Didn't seem you had anything with multiple declarations in your docs.